networkit/cpp/centrality/ApproxSpanningEdge.cpp

```diff
@@ -161,7 +161,7 @@ void ApproxSpanningEdge::run() {
 
             queue.push(source);
             toVisit[source] = NodeStatus::NOT_IN_COMPONENT;
-            count nVisited = 0;
+            [[maybe_unused]] count nVisited = 0;
```

Comment:
For better consistency with the rest of the codebase, can't we use `tlx::unused`? We could replace it with something else (e..g, `[[maybe_unused]]`) in the future.

Reply:
Hm, good point. Will exchange the occassions.